ccdproc/combiner.py

```diff
@@ -775,6 +779,10 @@ def combine(img_list, output_file=None,
                 ccd.mask[x:xend, y:yend] = comb_tile.mask
             if ccd.uncertainty is not None:
                 ccd.uncertainty.array[x:xend, y:yend] = comb_tile.uncertainty.array
+            # Clean up any open files
```

Comment:
The comment is now wrong. At this point there should only be no open file anymore

Reply:
Updating....